documentation/src/docs/metrics.md

```diff
@@ -78,16 +119,18 @@ public interface DynamicLogger {
 }
 ```
 
-- incCounterValue() : Increase Counter with given value.
-- updateCounterValue() : Updates the counter with given value.
-- freezeCounter() : Notifies that the counter will not be updated.
-- reportGaugeValue() : Reports Gauge value.
-- freezeGaugeValue() : Notifies that the gauge value will not be updated.
-- recordMeterEvents()  : Record the occurrence of a given number of events in Meter.
+- `incCounterValue()`: Increases the [Counter](https://metrics.dropwizard.io/3.1.0/manual/core/#counters) with the given value.
+- `updateCounterValue()`: Updates the [Counter](https://metrics.dropwizard.io/3.1.0/manual/core/#counters) with the given value.
+- `freezeCounter()`: Notifies that, the [Counter](https://metrics.dropwizard.io/3.1.0/manual/core/#counters) will not be updated.
+- `reportGaugeValue()`: Reports the [Gauge](https://metrics.dropwizard.io/3.1.0/manual/core/#gauges) value.
+- `freezeGaugeValue()`: Notifies that, the [Gauge](https://metrics.dropwizard.io/3.1.0/manual/core/#gauges) value will not be updated.
+- `recordMeterEvents()`: Records the occurrences of a given number of events in [Meter](https://metrics.dropwizard.io/3.1.0/manual/core/#meters).
+
+
+# Example for Starting a Metric Service
 
+This example is from file `io.pravega.segmentstore.server.host.ServiceStarter`. It starts Pravega Segment Store service and the Metrics Service is started as a sub service.
 
-# 2. Example for starting a Metric service
-This example is from file io.pravega.segmentstore.server.host.ServiceStarter. It starts Pravega SegmentStore service and a Metrics service is started as a sub service.
 ```java
 public final class ServiceStarter {
```

Comment:
Copy-pasted code. Let's put a reference to the actual code file.

Reply:
hyperlink provided :